bucket/siyuan-note.json

```diff
@@ -0,0 +1,47 @@
+{
+    "homepage": "https://github.com/siyuan-note/siyuan",
+    "description": "SiYuan is a Markdown Block-Reference and Bidirectional-Link note-taking application",
+    "version": "1.4.8",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://b3log.org/siyuan/eula.html"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win.exe#/siyuan.7z",
+            "hash": "2e72af2c6211595ce4e47c2141e029ce53fb7f934d383f9d8f79493ac6a70b61",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-64.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\" -Recurse"
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/siyuan-note/siyuan/releases/download/v1.4.8/siyuan-1.4.8-win32.exe#/siyuan.7z",
+            "hash": "7f97024325040287688ab0ea12ffd8b0e0045680e82c734461254e6e382fd53a",
+            "pre_install": [
+                "Expand-7zipArchive \"$dir\\`$PLUGINSDIR\\app-32.7z\" \"$dir\"",
+                "Remove-Item \"$dir\\`$*\" -Recurse"
+            ]
+        }
+    },
+    "shortcuts": [
+        [
+            "siyuan.exe",
+            "SiYuan Note"
+        ]
+    ],
+    "checkver": {
+        "url": "https://github.com/siyuan-note/siyuan/releases/latest",
+        "regex": "/tree/v([\\d.]+(-[\\w.]+)?)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://github.com/siyuan-note/siyuan/releases/download/v$version/siyuan-$version-win.exe#/siyuan.7z"
+            },
+            "32bit": {
+                "url": "https://github.com/siyuan-note/siyuan/releases/download/v$version/siyuan-$version-win32.exe#/siyuan.7z"
+            }
+        }
+    }
+}
```

Comment:
Newline missing at the end of the file

Reply:
Thanks for the good check!
But the my original fork has been deleted, I fix the issue in a new PR.
[Pull Request #7254 · ScoopInstaller/Extras](https://github.com/ScoopInstaller/Extras/pull/7254)